pkg/proto/agentbaker/v1/config.proto

```diff
@@ -225,4 +230,40 @@ message NetworkConfig {
   string cni_plugins_url = 7;
   string subnet = 8;
   string route_table = 9;
+}
+
+message CustomLinuxOSConfig {
+  bool should_config_swap_file = 1;
+  bool should_config_transparent_hugepage = 2;
+  bool should_config_containerd_ulimits = 3;
```

Comment:
The flags are likely artefact of not having a good mechanism to indicate unset field in shell? If we do have a clear mechanism with the switch to protobuf / JSON serialization - will hopefully not need these flags